regression/run_tests.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+converter_components=${1:-converter_components}
+script_dir=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+if ! command -v "$converter_components" > /dev/null; then
+  echo "$converter_components not found"
+  exit 1
+fi
+
+failures=()
+for FORMAT in "ASCII" "ABBREV_ASCII" "BINARY" "FLATTENED_BINARY" "JSON"; do
+    "$converter_components" "$script_dir/../database/messages_public.json" "$script_dir/BESTUTMBIN.GPS" $FORMAT > /dev/null
+    retval=$?
+    if [ $retval -ne 0 ]; then
+      echo "converter_components failed for $FORMAT failed with status $retval"
+    fi
+
+    if ! cmp -s "$script_dir/BESTUTMBIN.GPS.$FORMAT" "$script_dir/targets/BESTUTMBIN.GPS.$FORMAT"; then
+        failures+=("OEM-$FORMAT")
+    else
+      rm "$script_dir/BESTUTMBIN.GPS.$FORMAT"
```

Comment:
is there a reason lines 20 and 22 have different indentations?

Reply:
My bad. Thanks!